index.html

```diff
@@ -148,6 +148,35 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+        <!-- ==================  TEMPLATE  ================== -->
+        <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Angel Le</p>
+          <p class="contact">
+            <i class="fa-brands fa-facebook"></i>
+            <a href="https://www.facebook.com" target="_blank">https://www.facebook.com/angelxuanle</a>
```

Comment:
Uh oh! I think you intended for this link to go to your Facebook, but what you put in the `href` attribute only points to `https://www.facebook.com`.

To fix this, you just need to move what you put inside of your `<a>` tag into that `href` attribute, and then you're free to put whatever display text you want into the middle! 😃 

Here's an example that would fix this link:

```
<a href="https://www.facebook.com/angelxuanle" target="_blank">My super awesome Facebook profile!</a>
```

Reply:
I have made the change